src/services/cherry-picker.ts

```diff
@@ -156,6 +161,26 @@ export class CherryPicker {
     return node
   }
 
+  // Fetch app/node's service log from redis
+  async fetchRawServiceLogs(blockchain: string, rawNodeIDs: string[]): Promise<{ [id: string]: string }> {
+    const redisKeys = [] as string[]
+    const rawServiceLogs = {} as { [id: string]: string }
```

Comment:
```suggestion
    const rawServiceLogs: { [id: string]: string } = {} 
```

Reply:
`as` is better for when you're "casting" from one type to another, on this case as the  type is being defined is better to add next to the variable's name